### DISCUSSION TITLE
Codespaces menu is cut off in Firefox (working in Chromium)

### DISCUSSION THREAD

**Discussion by tippfehlr** [community/community#87321] [Codespaces]
### Select Topic Area

Bug

### Body

**Firefox:**
![image](https://github.com/community/community/assets/46620049/ef785256-19cb-480c-a5e0-b1222075a4e4)

**Chromium:**
![image](https://github.com/community/community/assets/46620049/f1635cd8-2e36-488e-801a-fd1d47a25e8a)

I can verify that the elements are there in devtools (on firefox).

**Comment by luctowers** [comment]
can confirm
